autogpt_platform/backend/backend/sdk/provider.py

```diff
@@ -83,7 +83,7 @@ def credentials_field(self, **kwargs) -> CredentialsMetaInput:
             "credentials_types": (
                 list(self.supported_auth_types)
                 if self.supported_auth_types
-                else ["api_key"]
+                else ["not_configured"]
```

Comment:
Why not just:
```suggestion
                else []
```
?

Then we also don't have to treat `"not_configured"` as if it's an auth type